mozilla-services/socorro
# fix bug 1440471, 1434425 - remove raw_update_channels and graphics_device related things

This removes the `raw_update_channels` table and friends. This also removes the friends of `graphics_device`--but not the `graphics_device` table. That will get moved to Django-land in bug 1457747.

To test:

1. check out master and run `make dockersetup`
2. check out this branch and do:
   ```
   $ docker/as_me.sh bash
   you@processor:/app$ alembic -c docker/config/alembic.ini upgrade heads
   ```
3. check to make sure everything got removed using psql and `\d` and `\df`
